store/sqlstore/role_store.go

```diff
@@ -167,7 +180,7 @@ func (s *SqlRoleStore) Get(roleId string) (*model.Role, error) {
 func (s *SqlRoleStore) GetAll() ([]*model.Role, error) {
 	var dbRoles []Role
```

Comment:
This applies to similar occurrences;
```suggestion
	dbRoles := []Role{}
```

Reply:
Right @isacikgoz. Did not notice that. Correcting it.